Dockerfile

```diff
@@ -1,12 +1,26 @@
+# Get qemu-user-static
+FROM alpine:3.9.2 as qemu
+RUN apk add --no-cache curl
+ARG QEMUVERSION=2.9.1
+ARG QEMUARCH
+
+SHELL ["/bin/ash", "-o", "pipefail", "-c"]
+
+RUN curl -fsSL https://github.com/multiarch/qemu-user-static/releases/download/v${QEMUVERSION}/qemu-${QEMUARCH}-static.tar.gz | tar zxvf - -C /usr/bin
+RUN chmod +x /usr/bin/qemu-*
+
 # Build node feature discovery
-FROM golang:1.10 as builder
+ARG IMAGEARCH
```

Comment:
I think this ARG needs to be specified at the top of the file, before the first `FROM`

Reply:
Done.